tests/test_main.py

```diff
@@ -281,7 +281,6 @@ def test_140_check_suppress_migrations(self):
         pylint_res = self.run_pylint(path_modules, extra_params)
         real_errors = pylint_res.linter.stats.by_msg
         expected_errors = {
-            "invalid-name": 1,
```

Comment:
Tell me more about this change, please

Reply:
`cr` was flagged by this lint because its length was less than 3 characters. Pylint v3 no longer does this. Related PR: https://github.com/pylint-dev/pylint/pull/8813

Reply:
I got it

Thank you!